File: Data Structures & Algorithms/missing-number/submission-0.java
class Solution {
    public int missingNumber(int[] nums) {
        int len = nums.length;
        int xor_array=0, xor_nums=0;
        for(int i=0;i<len;i++){
            xor_array ^= nums[i];
        }
        for(int i=1;i<=len;i++){
            xor_nums ^= i;
        }
        return xor_nums^xor_array;
    }
}

File: Data Structures & Algorithms/longest-increasing-subsequence/submission-0.py
class Solution:
    def lengthOfLIS(self, nums: List[int]) -> int:
        dp = [1]*len(nums)

        for i in range(len(nums)):
            for j in range(0,i):
                if nums[j] < nums[i]:
                    dp[i] = max(dp[j]+1, dp[i])

        return max(dp)

